react-native/react/images/folders/fa-folder-private-empty.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@cecileboucheron is this from font-awesome or is this a our own? If it is from font-awesome @MarcoPolo we already have that so we shouldn't need to have our own copy. If it is our own we likely shouldn't name this fa- as that's for font-awesome stuff